oragono.go

```diff
@@ -39,6 +39,13 @@ func getPassword() string {
 	return strings.TrimSpace(text)
 }
 
+func fileDoesNotExist(file string) bool {
```

Comment:
Hmm, might make sense to name this function `fileExists` and have it return the opposite? Then the if down there will just be `if fileExists(cert) || fileExists(key) {` which feels a bit easier to understand at a glance.

Reply:
The negation of this condition is actually "we don't know whether the file exists or not" --- `stat(2)` could either succeed, or could fail for some reason other than the file not existing.

Reply:
Ahh fair enough, that construction always confuses me a lil. All good then, makes sense.